app/models/area.rb

```diff
@@ -4,12 +4,14 @@ class Area < ActiveRecord::Base
 
   class << self
     def for_code(area_code)
+      return nil unless area_code.present?
```

Comment:
I think `unless object.present?` is just out of habit. But `if object.blank` is shorter and definitely clear. In the interest of merging this PR, I'd like to leave this one for now, but I'll try to use `if object.blank?` in the future.